Laravel.shibuya の行動規範を追加します

Fendo181 opened the pull request:
### 何を解決するのか

タイトル通りです。

行動規範を追加するにあたって以下の資料を参考させて頂きました。

- [code-of-conduct/LICENSE_ja.md at master · sf-jp-engineers/code-of-conduct](https://github.com/sf-jp-engineers/code-of-conduct/blob/master/LICENSE_ja.md)
- [builderscon/Code-of-Conduct.md at master · builderscon/builderscon](https://github.com/builderscon/builderscon/blob/master/Code-of-Conduct.md)

Fendo181 committed:
Laravel.shibuyaにおける行動規範を追加した

Fendo181 committed:
ライセンスを追加する

Fendo181 committed:
同じ説明が2つあったので修正する

YAhiru commented:
@Fendo181 
うおー！！ありがとうございます！！！基本的にはすごく良いと思います！(Lisenceの方は読んでません！)

なので僕的にはこのままマージしてもOKだと思います！

---

(以下は強いて意見を言うならです)

長谷川さんが[先日していたツイート](https://twitter.com/tomzoh/status/1208022367248584704)がいいなーと思っていて、(すでにある程度は書かれていますが)我々はみなさんにこう振舞ってもらいたいというポジティブな内容が推されているともっと良さそうな気がしました！

今日中には無理ですが、僕も内容考えてみますー

okashoi commented on README.md:
```diff
@@ -1,3 +1,34 @@
+- 他人の意見を尊重する
+- あおり、または侮辱、軽蔑的なコメントはしない
+- 容姿に関するコメントをしない
+- 意識的であろうと無意識であろうと差別的な発言はしない。誰か傷つく人がいないか考える
```
> 無意識であろうと差別的な発言はしない

これはちょっと矛盾していると思います（無意識であるのに、しないようにはできない）

それよりも「差別的な発言を受けたのであれば、相手に直接その旨を伝えるか、スタッフに伝えてください。（本人、あるいはスタッフから指摘を受けた人はそれを謙虚に受け止め、相手に謝罪をし、以降同じことを繰り返さないようにしてください」という運用のほうがあるべきかな、と思いましたー

（参考）このあたりの私の考えについてはこの辺で表明しております

https://twitter.com/okashoi/status/1208074775915941890

okashoi replied:
当然、考えたり気をつけることで事前に気づきを得ることもあるとは思うので、無意識のことに気をつけること自体は不可能だとは思います！

Fendo181 replied:
@okashoi 
ご提案ありがとうございます。
確かに、無意識に発言してしまうのを防ぐ事は難しいですね。


>差別的な発言を受けたのであれば、相手に直接その旨を伝えるか、スタッフにお伝えてください。

ここ、差別的な発言を受けた本人が直接、発言した人に指摘するには負担を追わせてしまうと考えている為スタッフに相談して下さいと追記しようと思います!

>仮に勉強会で上記にあげられている差別的なコメントを受けたのであればスタッフにお伝えてください。
スタッフから指摘を受けた人はそれを謙虚に受け止め、相手に謝罪をし、以降同じことを繰り返さないようにしてください

okashoi commented on README.md:
```diff
@@ -1,3 +1,34 @@
+
+などが挙げられます。
+
+職場で禁止されていることは基本的にここでも厳禁であり、これらのルールはソーシャル・メディアの発言や画像なども含みます。
```
> 職場で禁止されていることは基本的にここでも厳禁であり

「職場で禁止されていること」は職場によって異なることなので、あえて書く必要はないかなと思いました。

okashoi replied:
書くならば、具体的な行動の例をあげて書いたほうがいいかな、と思いました。

たとえば「本人の許諾なしに、顔が写った写真を SNS 等にアップロードしてはいけない」などでしょうか。

okashoi commented:
ありがとうございます！


長谷川さんのそのツイート、私もまったくもって同感ですねー

okashoi commented:
また、行動規範を設けるだけでなく運用についても考える必要もあるかな、と思います。

参考までに弊社で開催しているイベントではいまのところこんな運用を考えています。

https://github.com/hacker-s-gate/events/issues/44


これについては別途 issue たてておきますねー

#2

blue-goheimochi commented on README.md:
```diff
@@ -1,3 +1,34 @@
+勉強会の運営に携わるメンバーは安全に学べる場所を提供する努力を惜しみません。
+参加していただく全ての方にとっての安全な場作りのため、参加者、登壇者、スポンサー、ボランティア・スタッフ、主催者を含む全ての方に守っていただきたいルールがあります。
+
+ここはいかなる理由でもハラスメントがあってはならない場です。人種、年齢、性別、性自認、性嗜好、障碍の有無、出身国、滞在資格、民族や信仰する宗教による差別はあってはなりません。
```
個人的にはこの部分が、「人種、年齢、性別、性自認、性嗜好、障碍の有無、出身国、滞在資格、民族や信仰する宗教による差別」というのが限定的かなって感じが少ししました。（全てを羅列するのは不可能なのですが）

https://about.mercari.com/event-code-of-conduct/
メルカリの行動規範だと「その他の保護対象事項」という大きなくくりでも書いているのでそういうのも書いておけるとよいのでしょうか？

Fendo181 replied:
`その他の保護対象事項`を追記します!

後、メルカリの行動規範読んでみましたけど、良いですね！ 📝

blue-goheimochi reviewed (approved):
ありがとうございます！基本問題ないと思っています。

わたしもあひるさんが言っている、

> 我々はみなさんにこう振舞ってもらいたいというポジティブな内容が推されているともっと良さそうな気がしました！

は、すごくいいな！という感じがしていて、
その辺も含めて、みんなで考えつつ、少しずつ手を加えていけるとよいなと思いました。

https://phperkaigi.jp/coc.html

PHPerKaigiの行動規範はたしかに「ハラスメント」という言葉をあえて使っていなかったりしてますねー。

Fendo181 committed:
差別的な発言を受けた際の対応の文言を追加と、ソーシャル・メディアでの発言について文言を追加した

ref
- https://github.com/Laravel-shibuya/code-of-conduct/pull/1#pullrequestreview-335629421
- https://github.com/Laravel-shibuya/code-of-conduct/pull/1#discussion_r361234203

Fendo181 committed:
その他の保護対象事項を追加

ref:https://github.com/Laravel-shibuya/code-of-conduct/pull/1#discussion_r360682126

Fendo181 committed:
顔が写った写真に限定されてしまう為削除した

Fendo181 committed:
改行を追加

june29 commented on README.md:
```diff
@@ -5,20 +5,24 @@ Laravel.shibuyaはIRT(Interactive Round Table)を通じて直接相談や議論
 などが挙げられます。
 
-職場で禁止されていることは基本的にここでも厳禁であり、これらのルールはソーシャル・メディアの発言や画像なども含みます。
+仮に勉強会で上記にあげられている差別的なコメントを受けたのであればスタッフにお伝えてください。  
```
通りすがりの者です :pray:

>スタッフにお伝えてください

は

>スタッフにお伝えください

と書きたかったのかな？と感じました。

Fendo181 replied:
@june29 
コメントありがとうございます!

>スタッフにお伝えください
>と書きたかったのかな？と感じました。

そうです!ちょっと日本語おかしいので修正します 🙏

blue-goheimochi commented on README.md:
```diff
@@ -13,12 +13,16 @@ Laravel.shibuyaはIRT(Interactive Round Table)を通じて直接相談や議論
 - あおり、または侮辱、軽蔑的なコメントはしない
 - 容姿に関するコメントをしない
-- 意識的であろうと無意識であろうと差別的な発言はしない。誰か傷つく人がいないか考える
+- 差別的なコメントはしない。
```
ここは前後と統一して「。」はなしでよいかなと思います。

blue-goheimochi commented on README.md:
```diff
@@ -13,12 +13,16 @@ Laravel.shibuyaはIRT(Interactive Round Table)を通じて直接相談や議論
 
-職場で禁止されていることは基本的にここでも厳禁であり、これらのルールはソーシャル・メディアの発言や画像なども含みます。
+仮に勉強会で上記にあげられている差別的なコメントを受けたのであればスタッフにお伝えてください。
+スタッフから指摘を受けた人はそれを謙虚に受け止め、相手に謝罪をし、以降同じことを繰り返さないようにしてください
```
こちらは文末に「。」つけましょう！

Fendo181 committed:
お伝え下さい が正しい

refhttps://github.com/Laravel-shibuya/code-of-conduct/pull/1#pullrequestreview-336306488

Fendo181 committed:
句読点の統一

ref:https://github.com/Laravel-shibuya/code-of-conduct/pull/1#pullrequestreview-336308119

Fendo181 committed:
改行で表示されるように追加した

Fendo181 commented:
@okashoi @blue-goheimochi @YAhiru @june29 
レビューとコメントありがとうございました!
一旦この内容でマージしようと思います！

個人的に思った事ですが行動規範を書いてみて、Laravel.shibuyaに参加される方に安全に、楽しく学べる場所を提供しなければならないと意識するようになりました。

勉強会でやっていけない事を明文化するのは難しいのですが、[長谷川さんのツイート](https://twitter.com/tomzoh/status/1208022367248584704)から引用すると、自分が大切にしているコミュニティが仲良く楽しい場にしたい事を表明する上で、行動規範を書くのは大切だと感じました。

これらの内容は今後、ブログでも紹介したり、勉強会の最初の挨拶でアナウンスして、周知して行こうと思います!

改めて今後ともよろしくお願いします！ 🙇
